lib/orthoses/trace.rb

```diff
@@ -7,14 +7,15 @@ class Trace
     autoload :Method, 'orthoses/trace/method'
     autoload :Targetable, 'orthoses/trace/targetable'
 
-    def initialize(loader, patterns:)
+    def initialize(loader, patterns:, sort_union_types: true)
```

Comment:
I should have added this option in [the previous PR](https://github.com/ksss/orthoses/pull/79) 🙇